src/components/elements/text/text.tsx

```diff
@@ -87,7 +87,7 @@ export const Text = (props: IGenericElementProps<ITextSchema>) => {
 				// NOTE: Parent text body should be transformed into <div> to prevent validateDOMNesting error
 				{...(hasNestedFields() && { as: "div" })}
 			>
-				<Sanitize id={id}>{renderText()}</Sanitize>
+				<Sanitize sanitizeOptions={{ allowedAttributes: false }}>{renderText()}</Sanitize>
```

Comment:
what kind of attributes are expected for text tags?

Reply:
minimally, it has to allow `class` to get through, the class is needed by the styled-components. 
other than that it should allow any other attributes (e.g. `data-testid`) specified in the schema  to get through